heroku/cli
# migrate(W-14169290): spaces: Upgrade outbound-rules:remove

This PR moves `spaces:outbound-rules:remove` to the CLI and migrates to TypeScript

### Test Instructions
1. Checkout this branch and run `yarn && yarn build && cd packages/cli`
2. Run `./bin/run spaces:outbound-rules:remove RULE_NUM --space YOUR_SPACE --confirm YOUR_SPACE`
3. Check that the rule has been removed by running `heroku outbound-rules --space YOUR_SPACE`
